test/IdentityServer4.RavenDB.Storage.Tests/StoresTests/PersistedGrantStoreTests.cs

```diff
@@ -249,13 +248,6 @@ public async Task Store_should_update_record_if_key_already_exists()
             }
         }
         
-        private async Task<OperationalDocumentStoreHolder> GetOperationalDocumentStoreHolder_AndExecutePersistedGrantIndex()
-        {
-            var storeHolder = GetOperationalDocumentStoreHolder();
-            await ExecuteIndex(storeHolder.DocumentStore, new PersistedGrantIndex());
-            return storeHolder;
-        } 
-
         //[Fact]
         //public async Task GetAllAsync_is_implemented()
```

Comment:
@DejanMilicic Do we want to keep these tests? They are commented out so it looks like they are no longer valuable. Do you intend to refactor them and bring them back into the code base or should we remove them? If I am not mistaken, the retrieval and removal of all persisted grants using a specific filter is already tested above.

Reply:
@lukaszdobrzynski yes, persisted grants are already tested. Feel free to remove these commented out tests

Reply:
👍